spec/integ/crypto/verification.spec.ts

```diff
@@ -1604,7 +1604,7 @@ function sha256(commitmentStr: string): string {
     return encodeUnpaddedBase64(createHash("sha256").update(commitmentStr, "utf8").digest());
 }
 
-function encodeUnpaddedBase64(uint8Array: ArrayBuffer | Uint8Array): string {
+function encodeUnpaddedBase64(uint8Array: Uint8Array | Uint8ClampedArray<ArrayBuffer>): string {
```

Comment:
This is probably fine, but doesn't look immediately related to the task at hand. Can we break it out so it doesn't get lost in the sea of `defer` changes, please?

Ditto `buildReciprocateStartMessage`

Reply:
Sure, can split out the es2024 update to another PR

Reply:
Split out into https://github.com/matrix-org/matrix-js-sdk/pull/4832